### PR TITLE
Add metadata fields on files.

### PR DESCRIPTION
**Pull request by Pedro-vk**
## Description

Add metadata fields on files.

### CODE REVIEW

**Comment by Pedro-vk** [comment]
> These items are moved, so should disappear from `metadata.base`, including `size`

@kremalicious done! thanks :)

**Comment by kremalicious** [comment]
what's up with the build errors? Are they related to this PR change only, or are they generally failing right now cause all other PRs have failing builds too.

**Comment by Pedro-vk** [comment]
> what's up with the build errors? Are they related to this PR change only, or are they generally failing right now cause all other PRs have failing builds too.

I pushed a small change. And not totally sure why is failing, I'm checking it.

File: src/ddo/MetaData.ts [hunk 1]
```diff
@@ -1,104 +1,127 @@
...
+     * File URL.
+     * @type {string}
+     */
     url: string
+
+    /**
+     * File index.
+     * @type {number}
+     */
+    index?: number
```
Comment (kremalicious) [inline src/ddo/MetaData.ts]: `index` is a required attribute

Comment (Pedro-vk) [reply src/ddo/MetaData.ts]: Is not needed for squid API, because it is filled on `create` method.

**Pull request merged**